other/tools/run_protoc.sh

```diff
@@ -24,7 +24,11 @@ command -v protoc-gen-mavsdk > /dev/null || {
 }
 
 function snake_case_to_camel_case {
-    echo $1 | sed -r 's/(^|_)([a-z])/\U\2/g'
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        echo $1 | gsed -r 's/(^|_)([a-z])/\U\2/g'
```

Comment:
Does `gsed` exist on every Mac?

Reply:
No, it needs to be installed bye 'brew install gnu-sed'